### ISSUE TITLE
Improved parameter checks

### ISSUE THREAD

**Issue by noplanman** [diasporg/Poduptime#57]
I'm busy improving the parameter checks and have come across a few illogical things (bugs?).

- Is [this parameter](https://github.com/diasporg/Poduptime/blob/ca05819005b62813a29c44c1f50026f35a43b002/db/add.php#L20) really called `url`?
- In general, shouldn't the `pingdomurl` field be called `pingdom_api_key`?
- `adminkey` vs `token` [here](https://github.com/diasporg/Poduptime/blob/ca05819005b62813a29c44c1f50026f35a43b002/db/kill.php#L5)

Also, `db/add.php` is the only file that uses the `Logging` class to log issues. Should all files use it or really just this single one?

**Comment by dmorley** [comment]
Pingdom used to be a URL for stats, so people would put in a full url there 
that pingdom would generate, it included a guid so i needed it all. I at 
some point added uptimerobot and then i only nedded an api key to use their 
stats pull. Pingdom is no longer free, however i 'feel' like a good pod 
might still use them, i should validate that in the db and if untrue we 
could drop pingdom support overall.

In short yes these should both be updated to reflect todays reality, we 
should not allow new pods to be added using pingdoms stats anyways, no 
desire to scrape their pages for info.

**Comment by noplanman** [comment]
Ok, will adjust it like that.
Won't touch any DB columns yet, as that will be done in a fine sweep when we're ready 👍
